watch-parties/src/main/java/com/google/sps/servlets/ChatListerServlet.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.Date;
+import com.google.sps.data.Chat;
+
+
+@WebServlet("/chatstorage")
+public class ChatListerServlet extends HttpServlet {
+
+  public ArrayList<Chat> comments = new ArrayList<>();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // retrieve from datastore
+    Query query = new Query("Chat").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
Should we add a limit to the amount of chat messages we fetch? maybe limit it to the last 100 chat messages?